stories/css-shadow-variable-clash.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@compiled/react';
+
+export default {
+  title: 'css prop/variable clash',
+};
+
+const isPrimary = true;
+const Component1 = styled.p`
+  color: ${({ isPrimary }) => (isPrimary ? 'green' : 'red')};
+`;
+
+export const UsingComponent1 = (): JSX.Element => {
+  return <Component1 isPrimary={false}>Component1</Component1>;
+};
```

Comment:
Can you add 1 more example without props, eg
```
export const ComponentWithoutProps = (): JSX.Element => {
  return <Component> This should be red </Component>;
};
```

Reply:
Added another example without props